example/hello_world.py

```diff
@@ -167,15 +167,15 @@ def generate_with_se(sut, initial, environment, config, mission):
     sut = houston.ardu.ArduCopter(snapshot, config)
 
     # mission description
-    cmds = [
+    cmds = (
```

Comment:
Minor point: It should be possible to pass a list to the constructor; the constructor should transform the list into a tuple.